pysb/pattern.py

```diff
@@ -928,7 +928,7 @@ class ReactionPatternMatcher(object):
     [Rxn (reversible):
         Reactants: {'__s15': mSmac(b=None), '__s45': AMito(b=None)}
         Products: {'__s47': AMito(b=1) % mSmac(b=1)}
-        Rate: __s15*__s45*kf21 - __s47*kr21
+        Rate: kf21*__s15*__s45 - kr21*__s47
```

Comment:
Are commutative terms automatically and consistently sorted by sympy? I'm wondering if this new term ordering is stable.

Reply:
Yes, according to documentation ordering is automatic, consistent and arbitrary. See https://docs.sympy.org/1.5.1/tutorial/manipulation.html

> This because in SymPy, the arguments of the commutative operations Add and Mul are stored in an arbitrary (but consistent!) order, which is independent of the order inputted (if you’re worried about noncommutative multiplication, don’t be. In SymPy, you can create noncommutative Symbols using Symbol('A', commutative=False), and the order of multiplication for noncommutative Symbols is kept the same as the input). Furthermore, as we shall see in the next section, the printing order and the order in which things are stored internally need not be the same either.